package.json

```diff
@@ -18,13 +18,15 @@
   "jsdelivr": "dist/instantsearch.production.min.js",
   "unpkg": "dist/instantsearch.production.min.js",
   "sideEffects": false,
+  "types": "es/index.d.ts",
   "scripts": {
     "start": "yarn run dev",
     "dev": "yarn run storybook",
-    "build": "yarn run build:cjs && yarn run build:es && yarn run build:umd",
+    "build": "yarn run build:cjs && yarn run build:es && yarn build:types && yarn run build:umd",
     "build:umd": "BABEL_ENV=umd rollup --config scripts/rollup/config.js",
     "build:cjs": "BABEL_ENV=cjs babel src --extensions '.js,.ts,.tsx' --out-dir cjs/ --ignore 'src/index.es.ts','**/__tests__','**/__mocks__' --quiet",
     "build:es": "BABEL_ENV=es babel src --extensions '.js,.ts,.tsx' --out-dir es/ --ignore 'src/index.es.ts','**/__tests__','**/__mocks__' --quiet && BABEL_ENV=es babel src/index.es.ts --out-file es/index.js --quiet",
+    "build:types": "tsc -p tsconfig.declaration.json --outDir es/; mv es/index.es.d.ts es/index.d.ts",
```

Comment:
Does this mean the declaration is only for ESM, not for cjs?

Reply:
You cannot specify multiple declaration in `types` so we have to choose between CJS and ESM.

But I also noted that, even with the declaration files available, if you import `instantsearch.js` as CJS module you do not have any typing, do you confirm?